tests/snapshots/test_cli__short-help.snap

```diff
@@ -0,0 +1,39 @@
+---
+source: tests/test-cli.rs
+assertion_line: 81
+expression: stdout
+---
+cargo-vet 0.1.0
+Supply-chain security for Rust
+
+USAGE:
+    cargo-vet.exe [OPTIONS] [SUBCOMMAND]
+
+OPTIONS:
+        --all-features                 Activate all available features
+        --exclude <SPEC>               Exclude packages from being processed
+        --features <FEATURES>          Space-separated list of features to activate
+    -h, --help                         Print help information
+        --locked                       Do not pull in new "audits"
+        --log-file <LOG_FILE>          Instead of stderr, write logs to this file (only used after
+                                       successful CLI parsing)
+        --manifest-path <PATH>         Path to Cargo.toml
+        --no-default-features          Do not activate the `default` feature
+        --output-file <OUTPUT_FILE>    Instead of stdout, write output to this file
+    -p, --package <SPEC>               Package to process (see `cargo help pkgid`)
+    -V, --version                      Print version information
+        --verbose <VERBOSE>            How verbose logging should be (log level) [default: warn]
+                                       [possible values: off, error, warn, info, debug, trace]
+        --workspace                    Process all packages in the workspace
+
+SUBCOMMANDS:
+    accept-criteria-change    Accept changes that a foreign audits.toml made to their criteria
+    certify                   Mark `$package $version` as reviewed with `$message`
+    diff                      Yield a diff against the last reviewed version
+    fetch                     Fetch the source of `$package $version`
+    fmt                       Reformat all of vet's files (in case you hand-edited them)
+    forbid                    Mark `$package $version` as unacceptable with `$message`
```

Comment:
This isn't a thing. The terminology has changed to `violation`, and there's no CLI command for it: https://bholley.net/cargo-vet/design-choice-faq.html#why-is-there-no-helper-for-recording-violation-entries

Reply:
Ah yeah, I saw everything got changed to "violation" and fixed that, but wasn't sure about the command (hard to find the absence of something when skimming docs you're rereading for the 3rd time).

Minor question: how do you refer to a failure-as-a-result-of-violation? Before I could say "found a forbidden package" or something, "found a violated package" is... not right.

Reply:
"packaged marked as violating CRITERIA_FOO". "problematic package" would probably do in a pinch.